gateway/src/apicast/policy/find_service/path_based_finder.lua

```diff
@@ -17,7 +17,7 @@ function _M.find_service(config_store, host)
       if hosts[h] == host then
         local name = service.system_name or service.id
         ngx.log(ngx.DEBUG, 'service ', name, ' matched host ', hosts[h])
-        local matches = mapping_rules_matcher.matches(method, uri, {}, service.rules)
+        local matches = mapping_rules_matcher.matches(method, uri, ngx.req.get_uri_args(), service.rules)
```

Comment:
This is ok, but I would move this to the top. Calling to a function is expensive, and we should be worried about.

Do the same as method, uri, etc..

Reply:
Ok sure.